tools/ci_connector_ops/ci_connector_ops/pipelines/actions/builds.py

```diff
@@ -0,0 +1,40 @@
+#
+# Copyright (c) 2023 Airbyte, Inc., all rights reserved.
+#
+
+import tempfile
+from typing import List, Optional, Tuple
+
+import docker
+from ci_connector_ops.pipelines.contexts import ConnectorTestContext
+
+
+async def build_dev_image(context: ConnectorTestContext, exclude: Optional[List] = None) -> Tuple[str, str]:
```

Comment:
Personal preference, I think this function name is not descriptive enough. Perhaps calling out that this is specificially used to run the test suite might be better? I also think it is not very reusable outside of this test suite, so perhaps does not necessarily belong in the "actions" folder. 

Willing to debate on these points though!

Reply:
Sure, I originally put it in this module because it was generic enough to build any image, but I latter changed it to build connector images only for testing. Will move to `tests`.